SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialStaticComponentView/SpatialStaticComponentViewMock.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
```

Comment:
The following comment is for every new test class: according to `SpatialGDKTestGuidelines.h` the new test classes, defined in this PR, should be moved to the same folder as `ReferenceCountedLockingPolicyTest.cpp`, since they are relevant to this test file only.

Reply:
Happy to do that and apologies for missing this in the guidlines. However, going forward, if many classes want to mock these in an identical way - does it make sense to duplicate them everywhere or have in one place?

Reply:
That's a good point. In case that's exactly the same test code used fully in separate test files - we probably shouldn't duplicate it. However, I'd rather stick to Guidelines for now and put it with a test file. And then move or duplicate code in the future if needed.